MoviesMVVM/MoviesMVVM/DetailModule/View/FilmViewController.swift

```diff
@@ -306,14 +305,15 @@ final class FilmViewController: UIViewController {
         rateLabel.text = "\(data.rate)" + Constants.imdbFullRateString
         taglineLabel.text = "\(data.tagline)"
         descriptionLabel.text = data.overview
+
         genresLabel.text =
-            "\(data.genres.map(\.name).joined(separator: ", ")) \(Constants.dotChar) \((data.runtime) / 60)" +
+            "\((data.genres?.map(\.name).joined(separator: ", ")) ?? "") \(Constants.dotChar) \((data.runtime) / 60)" +
```

Comment:
```suggestion
            "\((data.genres?.map(\.name).joined(separator: ", ")) ?? "") \(Constants.dotChar) \((data.runtime) /)" +
```
что за магическое число 60, что делает?